tests/test_utils.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from pyboy.utils import WindowEvent
+
+
```

Comment:
I removed the initial test case, as it was testing internal functionality. I've corrected this by changing the code where the internal variable was erroneously used.